docs/3.0.0-beta.x/guides/queries.md

```diff
@@ -1,406 +1,287 @@
 # Queries
 
-Strapi provides a utility function `strapi.query` to make database queries ORM agnostic.
+Strapi provides a utility function `strapi.query` to make database queries.
 
-## Core Queries
+You can just call `strapi.query(modelName, pluginName)` to access the query API for any model.
 
-In Strapi's [core services](./services.md#core-services) you can see we call a `strapi.query` function.
+Those queries handle for you specific Strapi features like `groups` `filters` and `search`.
 
-When customizing your model services you might want to implement some custom database queries.
-To help you with that here is the current implementation of the queries for both `bookshelf` and `mongoose`.
+## API Reference
 
-You can simply copy and paste the code in your custom services.
+### `findOne`
 
-### Bookshelf
+This method returns the first entry matching some basic params.
+You can also pass a populate option to specify which relations you want to be populated.
+
+#### Examples
+
+**Find one by id**:
 
 ```js
-const _ = require('lodash');
-const { convertRestQueryParams, buildQuery } = require('strapi-utils');
-
-module.exports = ({ model }) => {
-  return {
-    find(params, populate) {
-      const withRelated =
-        populate ||
-        model.associations
-          .filter(ast => ast.autoPopulate !== false)
-          .map(ast => ast.alias);
-
-      const filters = convertRestQueryParams(params);
-
-      return model
-        .query(buildQuery({ model, filters }))
-        .fetchAll({ withRelated });
-    },
+strapi.query('post').findOne({ id: 1 });
+```
 
-    findOne(params, populate) {
-      const withRelated =
-        populate ||
-        model.associations
-          .filter(ast => ast.autoPopulate !== false)
-          .map(ast => ast.alias);
-
-      return model
-        .forge({
-          [model.primaryKey]: params[model.primaryKey] || params.id,
-        })
-        .fetch({
-          withRelated,
-        });
-    },
+**Find one by title**:
 
-    count(params = {}) {
-      const { where } = convertRestQueryParams(params);
+```js
+strapi.query('post').findOne({ title: 'my title' });
+```
 
-      return model.query(buildQuery({ model, filters: { where } })).count();
-    },
+**Find one by title and creation_date**:
 
-    async create(values) {
-      const relations = _.pick(
-        values,
-        model.associations.map(ast => ast.alias)
-      );
-      const data = _.omit(values, model.associations.map(ast => ast.alias));
+```js
+strapi
+  .query('post')
+  .findOne({ title: 'my title', created_at: '2019-01-01T00:00:00Z' });
```

Comment:
If think we need to specify here that the `created_at` field name depends on the db which can also be `createdAt` with Mongo

So here's my proposal

***Using bookshelf***
```js
strapi
  .query('post')
  .findOne({ title: 'my title', created_at: '2019-01-01T00:00:00Z' });
```
***Using mongoose***
```js
strapi
  .query('post')
  .findOne({ title: 'my title', createdAt: '2019-01-01T00:00:00Z' });
```

Reply:
And also the fact that these fields can be renamed by the user as well.